applications/FluidDynamicsApplication/symbolic_generation/navier_stokes/symbolic_stokes_cpp_template.cpp

```diff
@@ -198,6 +198,69 @@ void SymbolicStokes<TElementData>::AddBoundaryTraction(
     noalias(rRHS) += prod(rData.lhs,values);
 }
 
+template <class TElementData>
+void SymbolicStokes<TElementData>::Calculate(
+    const Variable<double> &rVariable,
+    double &rOutput,
+    const ProcessInfo &rCurrentProcessInfo)
+{
+    KRATOS_TRY
+    rOutput = 0.0;
+
+    if (rVariable == HEAT_FLUX) {
+        TElementData data;
+        data.Initialize(*this, rCurrentProcessInfo);
+        rOutput = 0.0;
+        // Shape functions
+        const auto &r_geometry = this->GetGeometry();
+
+        const unsigned int num_nodes = r_geometry.PointsNumber();
+        Vector data_N(num_nodes);
+        for (unsigned int i = 0; i < num_nodes; i++) {
+            data_N[i] = 1.0 / num_nodes;
```

Comment:
Doesn't this do integer division?
Usually the denominator needs to be static_casted to a double

Reply:
this is double by int, so wouldn't it be double? https://stackoverflow.com/questions/5563000/implicit-type-conversion-rules-in-c-operators

Reply:
hm ok if you checked
I thought I had seen this somewhere else, sorry for the noise

Reply:
e.g. here, this is relatively new code
https://github.com/KratosMultiphysics/Kratos/blob/c46c9420cd32d8557d8d3912c58a3c39fa34300a/kratos/utilities/normal_calculation_utils.cpp#L514

but might have been copy pasted ...

Reply:
I guess what you suggested is a good practice, so as to explicitly state the intention and definitely better than c-style caste https://www.stroustrup.com/bs_faq2.html#static-cast
I will make the change. Thank you.

Reply:
done